custom/abt/reports/supervisory_datasource_indicators.json

```diff
@@ -0,0 +1,225 @@
+[
```

Comment:
How are these indicator files used? You may be interested in `CustomDataSourceConfiguration`s which I think might be designed for this use case? Essentially a config file you keep in the repo that will automatically sync to a UCR table.